MIGRATION.md

```diff
@@ -0,0 +1,338 @@
+# Migration from 1.x to 2.0
+
+This document describes the main changes introduced in SDK `2.0` compared to `1.x`.
+
+### Product Modules 
+
+All the products (RUM, Trace, Logs, etc.) are now extracted into different modules. That allows you to integrate only what is needed into your application.
+
+In comparison with version 1.x where all products were in a single module `Datadog`, now you need to adopt the following libraries instead:
+
+- `DatadogCore`
+- `DatadogLogs`
+- `DatadogTrace`
+- `DatadogRUM`
+- `DatadogWebViewTracking`
+
+These come in addition to the existing `DatadogCrashReporting` and `DatadogObjc`.
+
+<details>
+  <summary>SPM</summary>
+
+  ```swift
+let package = Package(
+    ...
+    dependencies: [
+        .package(url: "https://github.com/DataDog/dd-sdk-ios", from: "2.0.0")
+    ],
+    targets: [
+        .target(
+            ...
+            dependencies: [
+                .product(name: "DatadogCore", package: "dd-sdk-ios"),
+                .product(name: "DatadogLogs", package: "dd-sdk-ios"),
+                .product(name: "DatadogTrace", package: "dd-sdk-ios"),
+                .product(name: "DatadogRUM", package: "dd-sdk-ios"),
+                .product(name: "DatadogCrashReporting", package: "dd-sdk-ios"),
+                .product(name: "DatadogWebViewTracking", package: "dd-sdk-ios"),
+            ]
+        ),
+    ]
+)
+  ```
+</details>
+
+<details>
+  <summary>CocoaPods</summary>
+
+  ```ruby
+  pod 'DatadogCore'
+  pod 'DatadogLogs'
+  pod 'DatadogTrace'
+  pod 'DatadogRUM'
+  pod 'DatadogCrashReporting'
+  pod 'DatadogWebViewTracking'
+  pod 'DatadogObjc'
+  ```
+</details>
+
+<details>
+  <summary>Carthage</summary>
+
+  The `Cartfile` stays the same: 
+  ```
+  github "DataDog/dd-sdk-ios"
+  ```
+
+  In Xcode, you **must** link the following frameworks:
+  ```
+  DatadogInternal.xcframework
+  DatadogCore.xcframework
+  ```
+
+  Then you can select the modules you want to use:
+  ```
+  DatadogLogs.xcframework
+  DatadogTrace.xcframework
+  DatadogRUM.xcframework
+  DatadogCrashReporting.xcframework + CrashReporter.xcframework
+  DatadogWebViewTracking.xcframework
+  DatadogObjc.xcframework
+  ```
+</details>
+
+**NOTE**: In case of Crash Reporting and WebView Tracking usage it's also needed to add RUM and/or Logs modules to be able to report events to RUM and/or Logs respectively.
```

Comment:
```suggestion
**Note**: In case of Crash Reporting and WebView Tracking usage it's also needed to add RUM and/or Logs modules to be able to report events to RUM and/or Logs respectively.
```